src/drivers/ports/esp32/can_port.i

```diff
@@ -270,6 +300,8 @@ int can_port_start(struct can_driver_t *self_p)
     volatile struct esp32_can_t *regs_p;
     struct pin_device_t *tx_pin_p;
     struct pin_device_t *rx_pin_p;
+    struct time_t now, stop, delta; ///< for time measurement in STATUS reg polling
```

Comment:
Remove the comment. Easy to understand what the variables are used for based on the code.